man/man8/zfs-rewrite.8

```diff
@@ -64,6 +66,20 @@ Print names of all successfully rewritten files.
 Don't cross file system mount points when recursing.
 .El
 .Sh NOTES
+Rewrite works by replacing an existing block with a new block of the same
+logical size.
+Changed dataset properties that operate on the data or metadata without
+changing the logical size will be applied.
+These include
+.Sy checksum ,
+.Sy compression ,
+.Sy dedup
+and
+.Sy copies .
+Changes to properties that do affect the size of a logical block, like
```

Comment:
I am not sure "do" here puts proper emphasis in the sentence, but if you say so.  I'd instead put "only" there.